pandas/core/generic.py

```diff
@@ -5303,10 +5332,10 @@ def _check_inplace_setting(self, value) -> bool_t:
         return True
 
     def _get_numeric_data(self):
-        return self._constructor(self._mgr.get_numeric_data()).__finalize__(self,)
+        return self._constructor(self._mgr.get_numeric_data()).__finalize__(self)
```

Comment:
hmm i think these have already been changed in master. can you avoid doing non-essential changes.

Reply:
Sorry I reverted these changes. Not sure how they were introduced. Probably black.